### ISSUE TITLE
Bytes-like object error on AWS

### ISSUE THREAD

**Issue by brantfaircloth** [ComparativeGenomicsToolkit/cactus#232]
I was hitting what looks like a Python 3 error on AWS with `Toil 3.24.0` when trying to run the evolver example w/ scaling (I'm testing prior to scaling up to larger analyses on AWS).

Switching to `Toil 4.1.0` seems to have fixed the issue, and the evolver example completed successfully (w/ scaling).  Reporting here as a potential workaround because the default instructions install `Toil 3.24.0`.

```
Using config from path /data/cactus-env/lib/python3.6/site-packages/cactus/cactus_progressive_config.xml.
xmlRoot = <multi_cactus inputSequenceNames="simHuman_chr6 simMouse_chr6 simRat_chr6 simCow_chr6 simDog_chr6" inputSequences="http://s3-us-west-2.amazonaws.com/jcarmstr-misc/testRegions/evolverMammals/simHuman.chr6 http://s3-us-west-2.amazonaws.com/jcarmstr-misc/testRegions/evolverMammals/simMouse.chr6 http://s3-us-west-2.amazonaws.com/jcarmstr-misc/testRegions/evolverMammals/simRat.chr6 http://s3-us-west-2.amazonaws.com/jcarmstr-misc/testRegions/evolverMammals/simCow.chr6 http://s3-us-west-2.amazonaws.com/jcarmstr-misc/testRegions/evolverMammals/simDog.chr6">
	<tree>((simHuman_chr6:0.144018,(simMouse_chr6:0.084509,simRat_chr6:0.091589)mr:0.271974)Anc1:0.020593,(simCow_chr6:0.18908,simDog_chr6:0.16303)Anc2:0.032898)Anc0;</tree>
	<cactus experiment_path="/tmp/tmpltp977r0/progressiveAlignment/Anc0/Anc0_experiment.xml" name="Anc0" />
	<cactus experiment_path="/tmp/tmpltp977r0/progressiveAlignment/Anc1/Anc1_experiment.xml" name="Anc1" />
	<cactus experiment_path="/tmp/tmpltp977r0/progressiveAlignment/mr/mr_experiment.xml" name="mr" />
	<cactus experiment_path="/tmp/tmpltp977r0/progressiveAlignment/Anc2/Anc2_experiment.xml" name="Anc2" />
</multi_cactus>
Limiting maxMemory to physically available memory (4137218048).
Limiting maxDisk to physically available disk (990635835392).
Running Toil version 3.24.0-de586251cb579bcb80eef435825cb3cedc202f52.
Using default docker registry of quay.io/ucsc_cgl as TOIL_DOCKER_REGISTRY is not set.
Got ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), trying again in 0s.
Got ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), trying again in 1s.
Got ('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer')), trying again in 1s.
Issued job 'RunCactusPreprocessorThenProgressiveDown' a1d227dc-d172-4002-b244-175f319b498e with job batch system ID: 0 and cores: 1, disk: 2.0 G, and memory: 3.3 G
Exception encountered in scaler thread. Making a best-effort attempt to keep going, but things may go wrong from now on.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/toil/provisioners/clusterScaler.py", line 885, in tryRun
    currentNodeCounts)
  File "/usr/local/lib/python3.6/dist-packages/toil/provisioners/clusterScaler.py", line 522, in getEstimatedNodeCounts
    desiredSize=estimatedNodeCount)
  File "/usr/local/lib/python3.6/dist-packages/toil/common.py", line 1243, in logClusterSize
    self.log("current_size '%s' %i" % (nodeType, currentSize))
  File "/usr/local/lib/python3.6/dist-packages/toil/common.py", line 1234, in log
    self.mtailProc.stdin.write(message + "\n")
TypeError: a bytes-like object is required, not 'str'
```

**Comment by glennhickey** [comment]
Thanks for the heads up.  I had issues with 4.0.0 but hadn't seen this one on 3.24.0.  Anyway, glad 4.1.0 is working for you (good timing too, as it only came out hours ago).  We'll switch Cactus over to it ASAP.

**Comment by brantfaircloth** [comment]
My pleasure! Thanks for all your work on cactus/hal/etc. 👍

**Comment by glennhickey** [comment]
Resolved by #233
